src/linux/mod.rs

```diff
@@ -33,11 +39,8 @@ type KeyStatesMap = HashMap<KeybdKey, bool>;
 
 static BUTTON_STATES: Lazy<Mutex<ButtonStatesMap>> =
     Lazy::new(|| Mutex::new(ButtonStatesMap::new()));
+static END_HANDLE_EVENTS: AtomicBool = AtomicBool::new(false);
```

Comment:
Maybe rename this to be positive? `HANDLE_EVENTS`?

Reply:
okay, but what about the problem on the windows side?